LEGO1/lego/sources/misc/legotypes.h

```diff
@@ -32,8 +32,8 @@ typedef char LegoS8;
 typedef unsigned char LegoU8;
 typedef short LegoS16;
 typedef unsigned short LegoU16;
-typedef long LegoS32;
-typedef unsigned long LegoU32;
+typedef int LegoS32;
+typedef unsigned int LegoU32;
```

Comment:
Let's add a bunch of static asserts at the bottom of this file for all custom Lego types.

Reply:
I added them under each type to make it easier to see what failed and that all of them are covered, and to make my life easier when adding them.